WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3565,17 +3567,18 @@ e_byzantium = {
 		capital = 560 # Cherson
 		
 		hellenic_pagan = 600
-		bosporan_faith = 200
+		bosporan_faith = 1000
```

Comment:
It has several Bosporan Holy Sites, and it was the birthplace of the Hypsistarian religion. You're DANG RIGHT it has more importance to Hypsistarians than it does Hellenics.